index.html

```diff
@@ -630,7 +630,43 @@ <h2>DID Documents</h2>
     </section>
 
     <section>
-      <h2>DID Resolvers and DID Resolution</h2>
+      <h2>DID Resolvers</h2>
+
+      <p>
+A <a>DID resolver</a> is a software and/or hardware component that takes a
+<a>DID</a> as input and produces a <a>conforming DID document</a> as output.
+This process is called <a>DID resolution</a> and often includes resolution input
+options and resolution output metadata.
```

Comment:
Suggest include input and output metadata in the definition. Phrasing "often includes" implies much more variability than is intended here. 

```suggestion
<a>DID</a> (and associated options) as input and produces a <a>conforming DID document</a> (and associated metadata) as output in a process called <a>DID resolution</a>.
```